lib/vagrant-packer-plugin/command.rb

```diff
@@ -106,9 +113,10 @@ def execute
         end
 
         puts "[packer-info] Packaging berkshelf recipes..."
-        # TODO - make it parametric
-        Berkshelf::Cli.start(["package",@params.cookbooks_url.split('/')[-1]])
 
+        Berkshelf::Cli.start(["package",@params.cookbooks_url.split('/')[-1]])
+        # TODO - consider also params.berksfile, but not working yet
```

Comment:
@enzor any idea why this is not working? I just added `"-b #{@params.berksfile}"` as an argument; it works on commandline; FTR, this would allow us to locate our `Vagrantfile` everywhere in a project and point to the location of `Berksfile`, using `config.packer_build.berksfile = "../Berksfile"` (for example)